db/revision_cache_interface.go

```diff
@@ -101,15 +101,17 @@ type RevisionDelta struct {
 	AttachmentDigests []string // Digests for all attachments present on ToRevID
 	ToChannels        base.Set // Full list of channels for the to revision
 	RevisionHistory   []string // Revision history from parent of ToRevID to source revID, in descending order
+	ToDeleted         bool     // Flag if ToRevID is a tombstone
 }
 
-func newRevCacheDelta(deltaBytes []byte, fromRevID string, toRevision DocumentRevision) *RevisionDelta {
+func newRevCacheDelta(deltaBytes []byte, fromRevID string, toRevision DocumentRevision, deleted bool) *RevisionDelta {
```

Comment:
It feels like deleted should be added as a property on DocumentRevision, but we could leave that as a Mercury enhancement. What do you think about filing a tracking ticket for that?

Reply:
I think Jacques tech spec covers this, but I'll check and file an issue if not.